docs/sources/guides/whats-new-in-v7-1.md

```diff
@@ -52,7 +52,7 @@ The new internal linking feature for Elasticsearch allows you to link to other d
 
 ## Transformations
 
-We have added a new **Merge on time** transform that can combine many time series or table results. Unlike the join transform this combines the result into one table even when the time values does not align / match.
+We have added a new **Merge on time** transform that can combine many time series or table results. Unlike the join transform this combines the result into one table even when the time values do not align / match.
```

Comment:
Add a comma - "Unline the join transform,"

Reply:
Hello @achatterjee-grafana , changes made as per the suggestion